lesson5/hometask_oefimenko/person.rb

```diff
@@ -0,0 +1,83 @@
+require 'yaml'
+
+module Person
+
+  PATH_TO_SERIALIZED = 'serialized'
+
+  module Factory
+
+    class PersonBase
+      attr_writer :fullname, :sex, :age
+
+      def initialize(label, block)
```

Comment:
PersonBase constructor is not supposed to be called from outside of DSL, so Person module method :person already transform block to proc and pass it as an object